docs/docsite/rst/dev_guide/ansible_markup.rst

```diff
@@ -0,0 +1,92 @@
+.. _ansible_markup:
+
+**************
+Ansible markup
+**************
+
+Ansible markup is a markup language used for documenting Ansible modules, plugins, and roles.
+It started as a basic markup language allowing basic formatting (bold, italics, code, horizontal line) and linking (URLs, hyperlinks, Ansible module references, RST references in the rendered documentation).
+In 2022 the markup language was extended by semantic elements for values, module/plugin options, return values, environment variables, and for referencing plugins.
```

Comment:
```suggestion
Ansible markup allows you to format and structure documentation for Ansible modules, plugins, and roles.
It lets you add basic formatting to text, such as bold, italics, code, and horizontal lines, as well as create various references, such as URLs, hyperlinks, Ansible module references, and RST references.
The Ansible markup language lets you apply semantic elements for values, module/plugin options, return values, environment variables, and for referencing plugins.
```

Reply:
The first one seems to read more clearly to me. I'd go for that.

Along with the detail of 2022, would it be at all helpful to mention the first Ansible version that featured semantic markup?